app/controllers/admin/subdomains_controller.rb

```diff
@@ -47,6 +47,19 @@ def invite_current_user_to_subdomain(subdomain)
     end
   end
 
+  def invite_current_super_user_to_subdomain(subdomain)
```

Comment:
@Pralish you don't need to make a new method here, `invite_current_user_to_subdomain` already exists and you just need to modify that method to assign the new permissions